sdk/storage/azure-storage-blob-cryptography/pom.xml

```diff
@@ -42,30 +42,49 @@
       <artifactId>azure-storage-blob</artifactId>
       <version>12.0.0-preview.4</version>
     </dependency>
-<!--    <dependency>-->
-<!--      <groupId>com.azure</groupId>-->
-<!--      <artifactId>azure-core-test</artifactId>-->
-<!--      <version>1.0.0-preview.4</version>-->
-<!--      <scope>test</scope>-->
-<!--    </dependency>-->
-<!--    <dependency>-->
-<!--      <groupId>com.azure</groupId>-->
-<!--      <artifactId>azure-core-http-netty</artifactId>-->
-<!--      <version>1.0.0-preview.4</version>-->
-<!--      <scope>test</scope>-->
-<!--    </dependency>-->
-<!--    <dependency>-->
-<!--      <groupId>org.slf4j</groupId>-->
-<!--      <artifactId>slf4j-simple</artifactId>-->
-<!--      <scope>test</scope>-->
-<!--    </dependency>-->
-<!--    &lt;!&ndash; https://mvnrepository.com/artifact/org.spockframework/spock-core &ndash;&gt;-->
-<!--    <dependency>-->
-<!--      <groupId>org.spockframework</groupId>-->
-<!--      <artifactId>spock-core</artifactId>-->
-<!--      <scope>test</scope>-->
-<!--    </dependency>-->
-<!--    <dependency>-->
+    <dependency>
```

Comment:
Since this is a new module, please add `module-info.java` as well.

Reply:
@srnagar Once this PR gets merged and approved, could you add the module-info.java file?

Reply:
Added module file